x-pack/legacy/plugins/siem/package.json

```diff
@@ -13,7 +13,8 @@
   "devDependencies": {
     "@types/lodash": "^4.14.110",
     "@types/js-yaml": "^3.12.1",
-    "@types/react-beautiful-dnd": "^11.0.4"
+    "@types/react-beautiful-dnd": "^11.0.4",
+    "@welldone-software/why-did-you-render": "^4.0.0"
```

Comment:
I would move this up to `x-pack/package.json` as we are suppose to keep our package.json within SIEM to a minimum as per the last time we talked to core team members.